specification/security/resource-manager/Microsoft.Security/preview/2015-06-01-preview/vulnerabilityAssessment.json

```diff
@@ -0,0 +1,397 @@
+{
+    "swagger": "2.0",
+    "info": {
+        "title": "Security Center",
+        "description": "API spec for Microsoft.Security (Azure Security Center) resource provider",
+        "version": "2015-06-01-preview"

```

Comment:
please use our new API verion, 2019-01-01-preview
when it will be stable you can bump it to 2019-01-01

Reply:
Is that mandatory?
All of our prod APIs are "2015-06-01-preview", Having this specific one to 2019 will make confusing workflow for us